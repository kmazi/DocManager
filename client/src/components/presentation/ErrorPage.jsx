import React from 'react';

const ErrorPage = () => (
  <div id="errorpage" className="container">
    <h4 className="center-align">Oops..</h4>
    <p>
      An error just occured; We are sorry about this.
    </p>
  </div>
);

export default ErrorPage;
