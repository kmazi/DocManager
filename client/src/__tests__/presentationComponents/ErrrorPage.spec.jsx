import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ErrorPage from '../../components/presentation/ErrorPage';

describe('The ErrorPage component:', () => {
  const enzymeWrapper = shallow(<ErrorPage />);
  test('should render the ErrorPage component correctly', () => {
    const component = enzymeWrapper;
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
