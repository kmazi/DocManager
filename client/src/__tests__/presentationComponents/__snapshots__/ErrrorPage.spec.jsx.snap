// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The ErrorPage component: should render the ErrorPage component correctly 1`] = `
<div
  className="container"
  id="errorpage"
>
  <h4
    className="center-align"
  >
    Oops..
  </h4>
  <p>
    An error just occured; We are sorry about this.
  </p>
</div>
`;
